aten/src/ATen/native/cuda/RangeFactories.cu

```diff
@@ -67,7 +67,6 @@ Tensor& linspace_cuda_out(Tensor& result, Scalar start, Scalar end, int64_t step
       scalar_t scalar_start = start.to<scalar_t>();
       scalar_t scalar_end = end.to<scalar_t>();
       float step = static_cast<float>(scalar_end - scalar_start) / (steps - 1);
```

Comment:
Can you please apply the same pattern here?

Reply:
Great idea! Fixed.